icons/arrow-right-square.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "git",
+    "diff",
+    "renamed",
```

Comment:
```suggestion
```

Not sure about these tags, they only apply to a very specific use case, let's keep this generic.